nano/test_common/ledger.cpp

```diff
@@ -1,12 +1,19 @@
 #include <nano/node/node.hpp>
 #include <nano/test_common/ledger.hpp>
 
-nano::test::context::ledger_context::ledger_context () :
+nano::test::context::ledger_context::ledger_context (std::deque<std::shared_ptr<nano::block>> && blocks) :
 	store_m{ nano::make_store (logger, nano::unique_path (), nano::dev::constants) },
-	ledger_m{ *store_m, stats_m, nano::dev::constants }
+	ledger_m{ *store_m, stats_m, nano::dev::constants },
+	blocks_m{ blocks }
```

Comment:
Since `blocks` is passed by `&&` you could also `std::move` from it here, but not really important in this test code.